src/02-common-principles.md

```diff
@@ -146,10 +147,13 @@ my_dataset/
     ...
 ```
 
-In this example **only the `rawdata` subfolder and all subfolders of `derivatives`
-need to be BIDS compliant** datasets. This specification does not prescribe 
-anything about the contents of `sourcedata` folders in the above example - nor 
-does it prescribe the `sourcedata`, `derivatives`, or `rawdata` folder names. 
+In this example **only the `rawdata` subfolder** needs to be a BIDS-compliant
+dataset.
+The subfolders of `derivatives` MAY be BIDS-compliant derivatives datasets
+(see [Non-compliant datasets][non-compliant-datasets] for further discussion).
+This specification does not prescribe anything about the contents of `sourcedata`
+folders in the above example - nor does it prescribe the `sourcedata`,
+`derivatives`, or `rawdata` folder names.
```

Comment:
The last part is a bit confusing to me:

> nor does it prescribe the `sourcedata`, `derivatives`, or `rawdata` folder names.

with that you mean that I can call my BIDS dataset root, e.g. `eeg_matchingpennies` ... or whatever - and it is valid BIDS? However, WITHIN `eeg_matchingpennies`, I am *not* free to give `sourcedata` any other name than `sourcedata`.

I think this could be clarified.

Reply:
No, I understand it as

```
-- whatevenyoucallthis
   |-- sourcedata_or_so
   |-- rawdata_in_bids
   `--pipeline_results_or_so
      |-- pipeline1_in_bids
      `-- pipeline2_in_bids
```
where the top level is up to you (e.g. it could be `D:`), and neither `sourcedata_or_so` nor `pipeline_results_or_so` are themselves according to BIDS. But both `rawdata_in_bids ` and each `pipelineX_in_bids` directories is a BIDS dataset. So BIDS only starts playing a role further down in the directory tree, and there are three (related) BIDS datasets here.

Reply:
I think that the example might already be more clear if it would be renamed like this
```
my_project/
  originaldata/
    ...
  rawdata/
    dataset_description.json
    participants.tsv
    sub-01/
    sub-02/
    ...
  my_results/
    pipeline_1/
    pipeline_2/
    ...
```
where only `rawdata`, `pipeline_1` and `pipeline_2` are organized according to BIDS.

Reply:
Yes, @robertoostenveld's interpretation is correct. I think part of the issue is that that context is missing from the diff.

Still, I'm happy to make any modifications (such as that suggested by Robert) which make that clearer.